src/bosh_openstack_cpi/lib/cloud/openstack/cloud.rb

```diff
@@ -1,4 +1,5 @@
 require 'common/common'
+require 'membrane'
```

Comment:
Looks good. Units and integration tests pass. We will slightly refactor the `require` statements in a follow-up commit (moving the 2 require statements to the `Bosh` module  in openstack.rb where all our `requires` live).